modules/Core/src/main/java/org/terasology/logic/inventory/CharacterInventorySystem.java

```diff
@@ -115,6 +115,7 @@ public void onDropItemRequest(DropItemRequest event, EntityRef character) {
             }
         }
 
+        pickupItem.getComponent(ItemComponent.class).pickUpTime = 2000;
```

Comment:
Oh,  something like this should probably live in the DropItemEvent?   Possibly the intention was to allow the character to provide some default value?  So that each character could customize how long the item stays unpickupable?  Otherwise,  I would expect setting a default amount of time to be set on an item's prefab (there is a common parent you could plunk the default value on to...  is it called "itemBase.prefab"?)